Resources/Private/Translations/en/Main.xlf

```diff
@@ -20,6 +20,12 @@
 			<trans-unit id="imageCropper__aspect-ratio-placeholder" xml:space="preserve">
 				<source>Choose an Aspect Ratio</source>
 			</trans-unit>
+			<trans-unit id="content.inspector.editors.masterPluginEditor.selectPlugin" xml:space="preserve">
+				<source>Select a Plugin</source>
```

Comment:
Just wondering, can't we use labels from the old UI here?

Reply:
Sure but yesterday someone mentioned that the dependencies should be low so I thought it is better to have the labels. But can change that.